tests/appsec/api_security/test_schemas.py

```diff
@@ -107,11 +107,18 @@ def test_request_method(self):
         assert self.request.status_code == 200
         assert schema
         assert isinstance(schema, list)
-        # There should have two parameters here, one for api_match_AS003, the other for 200
-        assert len(schema[0]) == 2
-        assert all(isinstance(v, list) for v in schema[0].values())
-        assert all(1 <= len(v) <= 2 for v in schema[0].values())
-        assert all(isinstance(v[0], int) for v in schema[0].values())
+        if context.library == "ruby" and context.weblog_variant == "sinatra14":
```

Comment:
I would like the test to be the same for all libraries and framework, to underline the minimal common ground for all of us and help other libraries to support the feature. So I would like you to remove that condition if you agree?

Reply:
I 'm also not a fan of `ifs` in tests, it'd be better to have a more "open" tests that all tracers passes. But so far, not a blocker for me, up to you to decide.